app/modules/main/templates/mdDataTableColumn.html

```diff
@@ -1,13 +1,12 @@
-<th class="column {{columnAlignClass}}" ng-click="clickHandler()">
+<th class="column {{columnAlignClass}}" ng-disabled="!isSortingEnabled()" md-ink-ripple ng-click="clickHandler()">
     <div layout="row">
         <span ng-if="isSortingEnabled() && isColumnRightAligned()">
```

Comment:
I think this is not correct. The disabled is for disabling the th click, while the ng-if check on the 'hover icon' is for including this icon in the DOM. Your change will make that the 'hover icon' will still appear even if the sorting is disabled.